src/EventStore.Core/Services/Transport/Grpc/Enumerators.AllSubscriptionFiltered.cs

```diff
@@ -241,7 +240,7 @@ await _channel.Writer.WriteAsync((new ResolvedEvent?(), position), ct)
 
 				_bus.Publish(new ClientMessage.FilteredSubscribeToStream(Guid.NewGuid(), _subscriptionId,
 					new ContinuationEnvelope(OnSubscriptionMessage, _semaphore, _cancellationToken), _subscriptionId,
-					string.Empty, _resolveLinks, _user, _eventFilter, (int)_checkpointIntervalMultiplier));
```

Comment:
_checkpointIntervalMultiplier doesn't appear to be used outside the constructor for calculating the checkpointInterval. maybe we can remove it?